DEVELOPMENT.md

```diff
@@ -17,19 +17,19 @@ cd docker
 Build the base image
 
 ```bash
-docker build base -t base
+docker build -t base -f base/Dockerfile .
 ```
 
 Build hub
 
 ```bash
-docker build hub -t hub
+docker build -t hub -f hub/Dockerfile .
 ```
 
 Build editor
 
 ```bash
-docker build editor -t editor
+docker build -t editor -f editor/Dockerfile .
```

Comment:
We actually are planning to move away from this context in an upcoming change. But it's ok to change it for now. Just fyi.

Reply:
OK, I just noticed that this didn't work (because of config/asound.conf in base/Dockerfile) and wanted to adjust the documentation before adding the same assumption about context with editor/unity-editor file.